pytest_github/plugin.py

```diff
@@ -4,14 +4,16 @@
 :license: MIT, see LICENSE for more details.
 """
 
-import os
 import logging
-import yaml
-import pytest
+import os
 import re
-import github3
 import warnings
+import yaml
+
+import pytest
 from _pytest.resultlog import generic_path
+import github3
```

Comment:
Should we ditch this import and follow the pattern below for importing `from github3.exceptions import GitHubError `?

Reply:
Just realized that we don't even need this:
`from github3.issues.issue import Issue`